packages/fxa-content-server/app/scripts/views/sign_in_unblock.js

```diff
@@ -52,16 +52,17 @@ const View = FormView.extend({
     );
   },
 
-  onSignInError(account, password, err) {
-    if (AuthErrors.is(err, 'INCORRECT_PASSWORD')) {
+  onSignInError(account, password, error) {
+    if (AuthErrors.is(error, 'INCORRECT_PASSWORD')) {
       // The user must go enter the correct password this time.
       this.navigate(this._getAuthPage(), {
+        account: this.getAccount(),
```

Comment:
This was needed to get signin unblock to go to the /signin page where the user could enter their password rather than to `/`